### PR TITLE
[Themes] Theme picker skip button

### LINKED ISSUE #10344: Update the theme picker Skip button when opened from app settings

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #10344. The PR fixes the theme picker Skip button functionality & hides it when the theme picker is opened from the settings.

**To test:**

Test 1
1. Start the store creation and navigate to the the picker
2. Notice the Skip button is visibe
3. Tap on the Skip button an verify the installation proceeds

Test 2
1. Go to Settings -> Tap on Themes
2. Notice the Skip button is not visible

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt [hunk 1]
```diff
@@ -28,11 +28,14 @@ class ThemePickerViewModel @Inject constructor(
...
     private val resourceProvider: ResourceProvider,
 ) : ScopedViewModel(savedStateHandle) {
-    private val _viewState = savedStateHandle.getStateFlow<ViewState>(viewModelScope, Loading)
-    val viewState = _viewState.asLiveData()
-
     private val navArgs: ThemePickerFragmentArgs by savedStateHandle.navArgs()
 
+    private val _viewState = savedStateHandle.getStateFlow(
+        scope = viewModelScope,
+        initialValue = ViewState(navArgs.isFromStoreCreation, Loading)
```
Comment (hafizrahman) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt]: np: I wonder if it helps code readability to name the arguments. Especially useful for `isSkipButtonVisible`, I think, to clarify why it matters to pass `isFromStoreCreation` for it:

```suggestion
        initialValue = ViewState(isSkipButtonVisible = navArgs.isFromStoreCreation, carouselState = Loading)
```

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt [hunk 2]
```diff
@@ -78,24 +85,30 @@ class ThemePickerViewModel @Inject constructor(
...
     fun onThemeTapped(themeUri: String) {
         triggerEvent(NavigateToThemePreview(themeUri, navArgs.isFromStoreCreation))
     }
 
-    sealed interface ViewState : Parcelable {
+    @Parcelize
+    data class ViewState(
+        val isSkipButtonVisible: Boolean,
+        val carouselState: CarouselState
+    ) : Parcelable
```
Comment (hafizrahman) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt:99]: I like the separation between ViewState and CarouselState here. 💯

**Review by hafizrahman — APPROVED** [review]
Left an np comment but otherwise this works correctly! Code looks good too 👍🏼

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt (hunk 1 above)
Comment (0nko) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt]: Yeah, that's a good idea 👍

**Comment by 0nko** [comment]
Thanks @hafizrahman!

**Pull request merged**